### DISCUSSION TITLE
ForServerToServer implementation

### DISCUSSION THREAD

**Discussion by KyiSer** [Jericho/ZoomNet#277] [Q&A] [answered]
Hello everyone,
I noticed that the version of the GitHub repository and the associated NuGet package are not the same. I wanted to start a discussion about this issue and get some input from the community. Have others experienced this same issue, and if so, how have you addressed it? Any suggestions or guidance on how to resolve this would be greatly appreciated. Thank you!

v0.58.0 in NuGet
![image](https://user-images.githubusercontent.com/33942455/228172904-49896776-5cfe-45cd-825c-34998b802671.png)

v0.58.0 in GitHub
![image](https://user-images.githubusercontent.com/33942455/228173268-4406d9ea-f3d8-4d6a-8e8d-fb778b5c4bc7.png)

I would like to use ForServerToServer, but unfortunately, I don't have that in v0.58.0

`<PackageReference Include="ZoomNet" Version="0.58.0" />`
![image](https://user-images.githubusercontent.com/33942455/228174518-4bd9cbd2-10c2-411f-93ad-f143769eb055.png)

**Comment by Jericho** [comment, marked as answer]
The code you reference from the GitHub repo is NOT the code in 0.58.0. You reference the `develop` branch which contains new features, code fix, etc. that may or may not have been published. If you want to look at the code for 0.58.0, you need to look for the [0.58.0 tag on the main branch](https://github.com/Jericho/ZoomNet/tree/0.58.0).

The `ForServerToServer` method you mention is a new feature [discussed here](https://github.com/Jericho/ZoomNet/issues/270) and which will be included in the [next release](https://github.com/Jericho/ZoomNet/milestone/69?closed=1).

**Reply by KyiSer** [reply]
Ohh I got it, thanks. I look forward to the next release
